### ISSUE TITLE
Error in `pointDTM_get()`

### ISSUE THREAD

**Issue by Nowosad** [kadyb/rgugik#10]
``` r
library(sf)
library(rgugik)
polygon_path = system.file("datasets/search_area.gpkg", package = "rgugik")
polygon = read_sf(polygon_path)
DTM = pointDTM_get(polygon, distance = 2) #error
#> 0/2
#> Warning in file(con, "r"): cannot open URL 'https://services.gugik.gov.pl/nmt/?
#> request=GetHByPointList&list=512868 360271,512868 360273,512868 360275,512868
#> 360277,512870 360265,512870 360267,512870 360269,512870 360271,512870
#> 360273,512870 360275,512870 360277,512872 360259,512872 360261,512872
#> 360263,512872 360265,512872 360267,512872 360269,512872 360271,512872
#> 360273,512872 360275,512872 360277,512874 360253,512874 360255,512874
#> 360257,512874 360259,512874 360261,512874 360263,512874 360265,512874
#> 360267,512874 360269,512874 360271,512874 360273,512874 360275,512874
#> 360277,512876 360247,512876 360249,512876 360251,512876 360253,512876
#> 360255,512876 360257,512876 360259,512876 360261,512876 360263,512876
#> 360265,512876 360267,512876 360269,512876 360271,512876 360273,512876
#> 360275,512876 360277,512876 360279,512878 360243,512878 360245,512878
#> 360247,512878 360249,512878 360251,512878 360253,512878 360255,512878
#> 360257,512878 360259,512878 360261,512878 360263,512878 360265,512878
#> 360267,512878 360269,512878 36 [... truncated]
#> Error in file(con, "r"): cannot open the connection to 'https://services.gugik.gov.pl/nmt/?request=GetHByPointList&list=512868 360271,512868 360273,512868 360275,512868 360277,512870 360265,512870 360267,512870 360269,512870 360271,512870 360273,512870 360275,512870 360277,512872 360259,512872 360261,512872 360263,512872 360265,512872 360267,512872 360269,512872 360271,512872 360273,512872 360275,512872 360277,512874 360253,512874 360255,512874 360257,512874 360259,512874 360261,512874 360263,512874 360265,512874 360267,512874 360269,512874 360271,512874 360273,512874 360275,512874 360277,512876 360247,512876 360249,512876 360251,512876 360253,512876 360255,512876 360257,512876 360259,512876 360261,512876 360263,512876 360265,512876 360267,512876 360269,512876 360271,512876 360273,512876 360275,512876 360277,512876 360279,512878 360243,512878 360245,512878 360247,512878 360249,512878 360251,512878 360253,512878 360255,512878 360257,512878 360259,512878 360261,512878 360263,512878 360265,512878 360267,512878 3
```

<sup>Created on 2020-08-09 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by Nowosad** [comment]
When I decreased the number of requests to 50 (it works when paste into the web browser):

```r
Error in file(con, "r") : 
  cannot open the connection to 'https://services.gugik.gov.pl/nmt/?request=GetHByPointList&list=512882 360247,512882 360249,512882 360251,512882 360253,512882 360255,512882 360257,512882 360259,512882 360261,512882 360263,512882 360265,512882 360267,512882 360269,512882 360271,512882 360273,512882 360275,512882 360277,512882 360279,512882 360281,512884 360225,512884 360227,512884 360229,512884 360231,512884 360233,512884 360235,512884 360237,512884 360239,512884 360241,512884 360243,512884 360245,512884 360247,512884 360249,512884 360251,512884 360253,512884 360255,512884 360257,512884 360259,512884 360261,512884 360263,512884 360265,512884 360267,512884 360269,512884 360271,512884 360273,512884 360275,512884 360277,512884 360279,512884 360281,512886 360219,512886 360221,512886 360223'
In addition: Warning message:
In file(con, "r") :
  cannot open URL 'https://services.gugik.gov.pl/nmt/?request=GetHByPointList&list=512882 360247,512882 360249,512882 360251,512882 360253,512882 360255,512882 360257,512882 360259,512882 360261,512882 360263,512882 360265,512882 360267,512882 360269,512882 360271,512882 360273,512882 360275,512882 360277,512882 360279,512882 360281,512884 360225,512884 360227,512884 360229,512884 360231,512884 360233,512884 360235,512884 360237,512884 360239,512884 360241,512884 360243,512884 360245,512884 360247,512884 360249,512884 360251,512884 360253,512884 360255,512884 360257,512884 360259,512884 360261,512884 360263,512884 360265,512884 360267,512884 360269,512884 360271,512884 360273,512884 360275,512884 360277,512884 360279,512884 360281,512886 360219,512886 360221,512886 360223': HTTP status was '400 Bad Request'
```

**Comment by kadyb** [comment]
Maybe this is a system related problem? These two examples work for me on Windows.
The problem is probably in this line: `str_output = readLines(prepared_URL, warn = FALSE)` (function `get_eval`).

The above example with 50 points works for me in R:
```r
prepared_URL = "https://services.gugik.gov.pl/nmt/?request=GetHByPointList&list=512882 360247,512882 360249,512882 360251,512882 360253,512882 360255,512882 360257,512882 360259,512882 360261,512882 360263,512882 360265,512882 360267,512882 360269,512882 360271,512882 360273,512882 360275,512882 360277,512882 360279,512882 360281,512884 360225,512884 360227,512884 360229,512884 360231,512884 360233,512884 360235,512884 360237,512884 360239,512884 360241,512884 360243,512884 360245,512884 360247,512884 360249,512884 360251,512884 360253,512884 360255,512884 360257,512884 360259,512884 360261,512884 360263,512884 360265,512884 360267,512884 360269,512884 360271,512884 360273,512884 360275,512884 360277,512884 360279,512884 360281,512886 360219,512886 360221,512886 360223"
debug: str_output = readLines(prepared_URL, warn = FALSE)
nchar(str_output)
#> 949
```

**Comment by Nowosad** [comment]
After some trial and error, I think the issue is with the space between coordinates. When I use `+` instead of ` `:

``` r
prepared_URL = "http://services.gugik.gov.pl/nmt/?request=GetHByPointList&list=512882+360247"
str_output = readLines(prepared_URL)
#> Warning in readLines(prepared_URL): incomplete final line found on 'http://
#> services.gugik.gov.pl/nmt/?request=GetHByPointList&list=512882+360247'
str_output
#> [1] "512882 360247 85.1"
```

<sup>Created on 2020-08-10 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by kadyb** [comment]
Please try fix from 39be8d7. I used `prepared_URL = gsub(" ", "%20", prepared_URL)` to replace characters. There is also the `URLencode` function from `utils`, but I didn't use it because it doesn't support Polish characters, and in the `geocode` function we also use gsub to replace characters. There is also the question of writing spaces with "%20" or "+" - [1](https://stackoverflow.com/questions/2678551/when-to-encode-space-to-plus-or-20), [2](https://stackoverflow.com/questions/13900835/the-origin-on-why-20-is-used-as-a-space-in-urls).

You should probably check if the rest of the Linux functions work (connection to URL step).

**Comment by Nowosad** [comment]
It seems ok.

(I check all of the functions yesterday - their examples seem to work fine)
